src/contributor-guide/contributing.md

```diff
@@ -95,6 +95,16 @@ The following list consists of questions you can ask to help determine the prope
 
 The number of 'Yes' answers should help you to determine the severity.
 
+### Pull request risk assessment:
+
+The risk label highlights the risk that the suggested changes may bring to the platform, it helps us to process pull requests, to decide on which version the pull requests should be delivered, it highlights pull requests that may need to be reviewed by other people, and it helps to decide whether a pull request should be approved or not depending on its risk.
```

Comment:
Sentence is too long - whole paragraph is one sentence
```suggestion
The risk label highlights the risk that the suggested changes may bring to the platform. It helps the Document Maintainer Team to decide on which version the pull requests should be delivered, which may need to be reviewed by other people and whether a request should be approved or not depending on its risk.
```

Reply:
Hi @BarnyShergold, thanks for reviewing, I've partially applied the suggestion, I've replaced `Document Maintainer Team` with `maintainers` which seems to be more appropriate in this case.